moderator.js

```diff
@@ -2,17 +2,19 @@ var lang = require('./languages');
 var bindings = require('./bindings');
 var secuence = require('./secuence');
 
-exports.getScriptFromDeck = function (deck) {
+exports.getScriptFromDeck = function (deck, selectedLanguage) {
```

Comment:
Wouldn't be a better idea to add lang.english as default for selected language? In that case you don't need two variables (language && selected language) to do the same. 

```exports.getScriptFromDeck = function (deck, selectedLanguage  = lang.english)```